src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class Rectangle extends Figure {
+    private double topSide;
+    private double lateralSide;
+    private double area;
```

Comment:
The 'area' variable should not be a field in this class. Instead, calculate the area on-demand in the getArea() method.